lib/smack/SmackInstGenerator.cpp

```diff
@@ -319,9 +319,9 @@ void SmackInstGenerator::visitInvokeInst(llvm::InvokeInst &ii) {
   if (f) {
     emit(rep->call(f, ii));
   } else {
-    // llvm_unreachable("Unexpected invoke instruction.");
-    SmackWarnings::warnUnsound("invoke instruction", currBlock, &ii,
-                               ii.getType()->isVoidTy());
+    llvm_unreachable("Unexpected invoke instruction.");
```

Comment:
What is the reason for this change?

Reply:
I think the original handling (i.e., `llvm_unreachable`) makes more sense since we want to be seriously warned (i.e., by a crash) when an invoke instruction invokes a non-function object.

Reply:
Ok, then please remove the comments. No need to leave them behind. We should also probably test this on SVCOMP one it gets merged.

Reply:
Done.